src/Altinn.Platform/Altinn.Platform.Storage/Storage/Helpers/InstanceHelper.cs

```diff
@@ -37,7 +37,9 @@ public static List<MessageBoxInstance> ConvertToMessageBoxInstance(List<Instance
                     Title = appTitles[instance.AppId].ContainsKey(language) ? appTitles[instance.AppId][language] : appTitles[instance.AppId]["nb"],
                     ProcessCurrentTask = instance.Process.CurrentTask,
                     AuthorizedForWrite = true,
-                    AllowDelete = true
+                    AllowDelete = true,
+                    AllowNewCopy = false,
+                    DeleteStatus = instance.InstanceState.IsDeleted ? "softDeleted" : "default",
```

Comment:
Should we follow a came casing with the status string? can't it just be lowercase?

Reply:
Jupp, I'll fix :)